### PR TITLE
#133 AdnWeightsDisplayNode deprecation 1.4.0

### PR DESCRIPTION
**Pull request by josevalverdeinbibo**
### Changes

- Added "Deprecated" section in release notes for 1.4.0 mentioning the AdnWeightsDisplayNode.
- Removed information exposed about the AdnWeightsDisplayNode in the paint tool section (Tools page). 
- Added a note about the AdnWeightsDisplayNode deprecation in the paint tool section (Tools page).

image

image

### CODE REVIEW

File: docs/maya/tools.md [hunk 1]
```diff
@@ -28,15 +28,10 @@ To open the tool:
...
-The painted weights are visualised in the viewport thanks to the **AdnWeightsDisplayNode**. This node is in charge of setting the vertex colours from the selected attribute in the UI on the mesh. This node is created and removed dynamically by the Paint Tool on entering and exiting from the context. Because of that, it is strongly recommended to keep the AdonisFX Paint Tool open while painting.
-
-<figure markdown>
-  ![AdnWeightsDisplayNode in the Node Editor](images/paint_tool_weights_display_node.png) 
-  <figcaption><b>Figure 2</b>: AdnWeightsDisplayNode created by the AdonisFX Paint Tool. </figcaption>
-</figure>
-
 > [!NOTE]
-> - If the painted weights are not displayed, it is probably because the node got removed. Please, select the transform node of an AdonisFX paintable deformer and click *Refresh From Selection* or restart the AdonisFX Paint Tool.
+> - With the optimizations introduced to the AdonisFX Paint Tool in version 1.4.0, the AdnWeightsDisplayNode is no longer needed and has been deprecated in this release.
```
Comment (josevalverdeinbibo) [inline docs/maya/tools.md:32]: Maybe is not necessary to expose here that AdnWeightsDisplayNode has been deprecated thanks to the optimizations introduced to the AdonisFX Paint Tool. Let me know what you think.

File: docs/release_notes.md [hunk 2]
```diff
@@ -37,6 +37,10 @@
 - Fixed a bug in the Volume Constraint that was causing flickering if the simulated geometry was far from the center of the world. *AdonisFX-1489*
 - Fixed the internal bounding box construction at start time to use the current point positions instead of the point positions at rest. *AdonisFX-1511*
 
+### Deprecated
+
+- The AdnWeightsDisplayNode has been deprecated. This deprecation is fully backward-compatible, ensuring that scenes created in earlier versions will work as expected in version 1.4.0. To safely remove deprecated nodes from your scenes, use the utility available in AdonisFX Menu > Utils > *Upgrade Deprecated Nodes*.
```
Comment (carlosmonteagudoinbibo) [inline docs/release_notes.md:42]: As this page contains the release notes, it is not a page to explain "how to" use a too. I would rewrite this slightly:
```suggestion
- The AdnWeightsDisplayNode is deprecated. This deprecation is fully backward-compatible, ensuring that scenes created in earlier versions will work as expected in version 1.4.0. To safely remove deprecated nodes from your scenes, a utility is added in AdonisFX Menu > Utils > *Upgrade Deprecated Nodes*.
```

File: docs/maya/tools.md (hunk 1 above)
Comment (carlosmonteagudoinbibo) [reply docs/maya/tools.md:32]: I think it is ok! Just a small suggestion
```suggestion
> - With the optimizations introduced to the AdonisFX Paint Tool in version 1.4.0, the AdnWeightsDisplayNode is deprecated and no longer needed.
```

**Review by carlosmonteagudoinbibo — CHANGES_REQUESTED** [review]

File: docs/maya/tools.md (hunk 1 above)
Comment (josevalverdeinbibo) [reply docs/maya/tools.md:32]: Done!

**Review by lucasferryinbibo — APPROVED** [review]
Agree with Carlos' suggestions

**Review by carlosmonteagudoinbibo — APPROVED** [review]

**Pull request merged**